kratos/geometries/line_3d_2.h

```diff
@@ -314,9 +314,9 @@ class Line3D2 : public Geometry<TPointType>
         const double lx = point0.X() - point1.X();
```

Comment:
you could write this code shorter as

    array_1d<double,3> lvec = point1 - point0;
    return norm_2(lvec);